app/controllers/articles_controller.rb

```diff
@@ -2,14 +2,23 @@
 
 # Controller to show, add, edit, update and delete articles
 class ArticlesController < ApplicationController
-
+  
   def index
     @articles = Article.all
     @users = User.all
+    if user_signed_in?
+      @user_now = User.find(current_user.id)
+    end
   end
-
+  
   def show
     @article = Article.find(params[:id])
+    @user = User.find(@article.owner)
+    @rel = @user.followers.find_by(follower: current_user)
+    
+    if user_signed_in?
+      @current_user_owner_article = true if @article.owner == current_user.id
```

Comment:
This could be done in a separate method

Reply:
All changes were applied in the following commits of this same PR